.github/workflows/lint.yaml

```diff
@@ -17,3 +17,6 @@ jobs:
           files: .
           config_file: .markdownlint.yaml
           dot: true
+
+      - name: typos-action spellchecker
```

Comment:
I noticed one pushed the jobs name is markdownlint (line 9)

So the CI is reporting "markdownlint" succeed for markdownlint+typos

do you want me to rename "markdownlint" to something else ?

or do you want me to move the typos action in its own jobs (but then we will have to checkout again)

Reply:
`markdownlint+typos` is fine. If you're allowed to use `+` I actually don't know off the top of my ahead, but any delimiter as long as we are consistent is fine with me. 
 
Thanks

Reply:
I updated the jobs to lint, but I'm currious about your feedbacks

Reply:
That is fine. Resolving.

Reply:
arf, I missed your comment @dylanhitt 

but I'm afraid 

```
Job name markdownlint+typos is invalid. Names must start with a letter or '_' and contain only alphanumeric characters, '-', or '_'
```